ksqldb-rest-app/src/test/java/io/confluent/ksql/api/BaseApiTest.java

```diff
@@ -282,6 +268,26 @@ protected void sendPostRequest(
     requestSender.accept(client.post(uri));
   }
 
+  protected void waitForQueryPublisherSubscribed() {
+      final Set<TestQueryPublisher> queryPublishers = testEndpoints.getQueryPublishers();
+      assertThat(queryPublishers, hasSize(1));
+      queryPublishers.forEach(
+          publisher -> {
+              final long timeout = System.currentTimeMillis() + 30_000L;
+              while (publisher.getSubscriber() == null) {
+                  try {
+                      Thread.sleep(100L);
+                  } catch (InterruptedException swallow) {
+                  }
```

Comment:
nit: I'd put the closing `{` just after the opening `{`

Reply:
Copy-paste from existing code :) I'm not opposed, can do that next time.